### DISCUSSION TITLE
Call for testers 🐛 (free credits on offer)

### DISCUSSION THREAD

**Discussion by kiwicopple** [supabase/supabase#19169] [General]
We’ve been working on a secret feature and we need testers. This is absolutely not for production, it’s in the TEST phase.

If you want to help us test, we’ll give you a direct line to the Supabase team to give your feedback (a slack channel). Some criteria:

- You must be signed up to [Fly.io](http://fly.io/)
- You must know how to read some [Fly docs](https://fly.io/docs/flyctl/extensions-supabase/)
- You should not put any data which isn’t already backed up

In return, we will give you 6 months of Supabase Pro Plan (in credits). We can only accept 10 testers right now and we will open up more testing slots each week.

Just drop a comment with your OrgID as a response here and I'll follow up with more details

--- 

edit 2023-11-27: inviting first 5 testers
edit 2023-11-28: inviting next 5 testers

**Comment by bahodirr** [comment]
OrgId: hflahgdlyqyablbumzbk

**Reply by kiwicopple** [reply]
You're tester # 1.

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Comment by remorses** [comment]
urtkranmtzcwhorketkm

I have been using fly.io for both notaku.so and holocron.so

**Reply by kiwicopple** [reply]
You're tester # 2.

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Comment by himeshp** [comment]
is it supabase `OrgId`?

**Reply by jk1551** [reply]
Yeah I believe so, if you can't find it in the dashboard you can use the supabase CLI to get it with "supabase orgs list"

**Reply by himeshp** [reply]
long-coffee-alpaca  │ inkryptis

**Reply by kiwicopple** [reply]
You're tester # 3 @himeshp 

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Comment by jk1551** [comment]
OrgId: qcskpcfcilwscfszeboo

**Reply by kiwicopple** [reply]
You're tester # 4.

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Comment by suciptoid** [comment]
OrgID: rbhvudlcjwcopczwrizc

**Reply by kiwicopple** [reply]
You're tester # 5.

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Reply by suciptoid** [reply]
Sorry I can't DM because not verified on Twitter 😅. Can I DM @supabase twitter account instead?

**Reply by kiwicopple** [reply]
I've got your twitter - will dm you

**Comment by gijigae** [comment]
Happy to participate as a tester.
OrgID: ixdeenqtwpvmuxbimvyu

**Reply by kiwicopple** [reply]
You're tester # 6.

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Comment by asrvd** [comment]
would love to help, been a supabase user since a long time <3

orgId: sinmimhyobssispugcan

**Reply by kiwicopple** [reply]
You're tester # 7.

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Reply by asrvd** [reply]
i'm not verified on twt, can you shoot me a dm instead on [@_asheeshh](https://twitter.com/_asheeshh), thanks :)

**Reply by asrvd** [reply]
> You're tester # 7.
> 
> Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:
> 
> - Your email address (so that I can invite you to a private slack channel)
> - Your Fly Org ID (if you have one)
> 
> Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel 

👀

**Comment by HarryET** [comment]
Happy to test! and excited to see whats going on with fly.io

OrgId: `yrskagqssuawnjxzgrgb`

**Reply by kiwicopple** [reply]
Hey again Harry 👋. You're tester # 8.

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Comment by alleneubank** [comment]
Sign us up please!

wxpfmwbhxqcjvjsyimfs

**Reply by kiwicopple** [reply]
You're tester # 9.

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Comment by MarceloPrado** [comment]
Would love to help and share feedbacks
OrgId: `zkvshroyeudpsdhvqypm`

**Reply by kiwicopple** [reply]
You're tester # 10.

Thanks! Can you please send me a [DM on twitter](https://twitter.com/kiwicopple) with:

- Your email address (so that I can invite you to a private slack channel)
- Your Fly Org ID (if you have one)

Make sure you link to this message so that I know who you are. We'll have the full details + instructions in the slack channel

**Comment by thedevdavid** [comment]
orgid: `qrafvimaivgmojztarzu`

**Comment by Rohithgilla12** [comment]
Org id `optimistic-ivory-eagle` 😄

**Comment by DenisVCode** [comment]
Org id `empty-aqua-bwqz0u2`

**Comment by benclmnt** [comment]
org id is `light-teal-puma`

**Comment by hipdev** [comment]
Org id `cbupuisdwtjxldrhppbp`
Edge db is cooking 👨‍🍳

**Comment by NikolayS** [comment]
👋 conscious-gray-beaver

**Comment by tannal** [comment]
Org id `cbupuisdwtjxldrhppbp`
happy to help

**Comment by MukeshKumarBagaria** [comment]
Org id - `xwjftxgjdrkfsgbebxek`
Twitter - [@Mukeshk82257521](https://twitter.com/Mukeshk82257521)
I am really interested in this opportunity can do manual as well as automated testing, Having exp. of deploying my node.js apps with `fly.io` 

Please reach out to me!

**Comment by trentguillory** [comment]
Org ID: `alive-cyan-xerinae` 

Very interested in helping test!

**Comment by ApexCyberPunk** [comment]
Still need testers?
